alacritty_terminal/src/config/mod.rs

```diff
@@ -242,20 +243,47 @@ impl Default for EscapeChars {
 }
 
 #[serde(default)]
-#[derive(Deserialize, Copy, Clone, Debug, Default, PartialEq, Eq)]
+#[derive(Deserialize, Copy, Clone, Debug, PartialEq)]
 pub struct Cursor {
     #[serde(deserialize_with = "failure_default")]
     pub style: CursorStyle,
     #[serde(deserialize_with = "option_explicit_none")]
     pub vi_mode_style: Option<CursorStyle>,
+    #[serde(deserialize_with = "default_thickness")]
```

Comment:
I think you need to add a `default = default_thickness` annotation otherwise it would print an error when the key is missing?

Reply:
It seems to work just fine?